internal/cephfs/util.go

```diff
@@ -97,3 +124,50 @@ func (cs *ControllerServer) validateExpandVolumeRequest(req *csi.ControllerExpan
 
 	return nil
 }
+
+// cephfsSnapshot represents a CSI snapshot and its cluster information.
+type cephfsSnapshot struct {
```

Comment:
can this be removed to Snapshot{} as all structs and functions inside cephfs folder already belong to cephfs.